ntbs-service/Models/NtbsContext.cs

```diff
@@ -35,11 +35,6 @@ public NtbsContext(DbContextOptions<NtbsContext> options)
         public virtual DbSet<Site> Site { get; set; }
         public virtual DbSet<Region> Region { get; set; }
         public virtual DbSet<Sex> Sex { get; set; }
-        public virtual DbSet<Episode> Episode { get; set; }
```

Comment:
One cannot define db sets for owned models as they don't have their own id - these weren't used anywhere